pkg/apis/io/v1alpha1/types.go

```diff
@@ -110,6 +110,7 @@ type JaegerAgentSpec struct {
 // JaegerStorageSpec defines the common storage options to be used for the query and collector
 type JaegerStorageSpec struct {
 	Type                  string                          `json:"type"` // can be `memory` (default), `cassandra`, `elasticsearch`, `kafka` or `managed`
+	Secrets               v1.SecretList                   `json:"secrets"`
```

Comment:
Just wondering whether this needs to be anything more than a list of strings? It is not actually defining the secret here, just a name of a secret defined externally?